addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -35,6 +35,35 @@
     @extend %o-we-background-layer;
 
     @each $module, $shapes in $o-bg-shapes {
+        @each $shape, $style in $shapes {
+            $url-params: '';
+            $colors: map-get($style, 'colors');
+            @each $i in $colors {
+                $cc: 5 - $i;
+                @if ($cc == 0) {
+                    $cc: 5;
+                }
+                // %23 is the url-encoded form of '#' which doesn't work as is in urls.
+                $color: str-replace("#{color('o-cc#{$cc}-bg')}", '#', '%23');
+                $url-params: '#{$url-params}&c#{$i}=#{$color}';
+            }
+
+            // eg: o_website_shape_bg_1_cc
+            &.o_#{$module}_#{str-replace($shape, '/', '_')}_cc {
```

Comment:
So this will also be only one rule (with two selectors) for old and new class

Reply:
Not sure what you mean with this part

Reply:
Mmh indeed. So forget about this comment, the rest still applies though

Reply:
You forgot to forget about this comment :grimacing: